### PR TITLE
Feat/user bubbles

### LINKED ISSUE #61: Feature/add user bubbles
## Description
Add user bubbles to show the list of users in the room. 

Reqs:
1. When user joins, the user bubble gets added to all clients in that room
2. When user joins, the joined user gets a list of all connected clients
3. When user leaves, the user bubbles on all connected clients get updated to remove that user
4. if there are more than 6 clients, the 7th bubble is a concatenation that says +1 or +2 as required
5. Add tooltip to show the name of the user

Example:
image

### PR DESCRIPTION
**Pull request by nmpereira**
## Description
Add user bubbles to show the list of users in the current room

To be closed together with: https://github.com/CommunityFocus/cf-backend/pull/52

Functionality:
1. When user joins, the user bubble gets added to all clients in that room
2. When user joins, the joined user gets a list of all connected clients
3. When user leaves, the user bubbles on all connected clients get updated to remove that user
4. if there are more than 6 clients, the 7th bubble is a concatenation that says +1 or +2 as required
5. Add tooltip to show the name of the user

Closes CommunityFocus/CommunityFocus/issues/61

1 user:
image

6+ users:
image

Tooltip:
image

### CODE REVIEW

File: src/components/UserBubbles/MultiUserBubble.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Tooltip } from "react-tooltip";
+import { TooltipContainer, UserBubbleStyled } from "./UserBubble.styled";
+
+const MultiUserBubble = (props: { users: string[] }) => {
+	const { users } = props;
+	// first and last character of user name
```
Comment (nmpereira) [inline src/components/UserBubbles/MultiUserBubble.tsx]:
```suggestion
```

File: src/components/UserBubbles/MultiUserBubble.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	return (
+		<TooltipContainer>
+			<UserBubbleStyled
+				$borderColor="brown"
+				data-tooltip-id="my-tooltip-multilin"
+				data-tooltip-html={users.join("<br /><br />")}
```
Comment (nmpereira) [inline src/components/UserBubbles/MultiUserBubble.tsx]:
```suggestion
				data-tooltip-html={users.join("<br />")}
```

File: src/components/UserBubbles/UserBubble.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import { Tooltip } from "react-tooltip";
+import { TooltipContainer, UserBubbleStyled } from "./UserBubble.styled";
+
+const UserBubble = (props: { user: string }) => {
+	const { user } = props;
+	// first and last character of user name
```
Comment (nmpereira) [inline src/components/UserBubbles/UserBubble.tsx]:
```suggestion
```

File: src/components/UserBubbles/UserBubbles.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+const UserBubbles = (props: { userListInRoom: string[] }): JSX.Element => {
+	const { userListInRoom } = props;
+
+	return (
+		<UserBubblesContainer>
+			{/* {userListInRoom.map((user: string) => (
+				<UserBubble user={user} key={user} />
+			))} */}
+
+			{/* if there are less than 7, show all otherwise dont render the user bubble, but show an extra bubble with the number not shown as the user */}
```
Comment (nmpereira) [inline src/components/UserBubbles/UserBubbles.tsx]:
```suggestion
```

File: src/components/Room.tsx [hunk 5]
```diff
@@ -49,8 +51,15 @@ const Room = (props: { globalUsersConnected: number }): JSX.Element => {
...
 	};
 
-	const onUsersInRoom = (value: string): void => {
-		setUsersInRoom(parseInt(value));
+	const onUsersInRoom = ({
+		numUsers,
+		userList,
+	}: {
+		numUsers: number;
+		userList: string[];
```
Comment (riddlew) [inline src/components/Room.tsx]: Can we extract this to its own type to make this a little more clean?

**Review by riddlew — CHANGES_REQUESTED** [review]

**Review by riddlew — APPROVED** [review]

**Pull request merged**